### ISSUE TITLE
Change h2 to h1 in out toolbar

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#5600]
**Current behavior**
Screenreader do not recognise that our title is important because it is not a h1 but a h2

<img width="454" height="107" alt="Image" src="https://github.com/user-attachments/assets/1c9cdbd5-d20a-47ab-a865-a730e3291642" />

**Wanted behavior**
The title should be a h1 everywhere but the style should stay the same. 
So the correct classes should be created and added to the different h2's

**Additional info:**
There are a lot of places where this title is used
